x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/inference/persistence/TrainedModelProvider.java

```diff
@@ -306,6 +334,14 @@ public void storeTrainedModelDefinitionDoc(
     }
 
     public void storeTrainedModelMetadata(TrainedModelMetadata trainedModelMetadata, ActionListener<Void> listener) {
+        storeTrainedModelMetadata(trainedModelMetadata, listener, false);
+    }
+
+    public void storeTrainedModelMetadata(
```

Comment:
This change isn't really necessary because no one outside of the function above calls this method. I can revert the change but I figured it it'd make it consistent with the other `store*` entry points.